README.md

```diff
@@ -0,0 +1,65 @@
+
+# Project Name
+
+> this project helps you keep track of reserved missions and rockets on ypor profile add or remove the on your profile.
+
+
+## Built With
+
+- react
+- bootstraps
+- redux
+
+**To get this project locally**
+
+## Run the following command on your terminal
+
+
+```
+git clone https://github.com/businesskaniki/spacex.git
+
+```
+```
+cd spacex
+
+```
+```
+npm install
+
+```
+```
+npm start
+
+```
+
+## Authors
+
+👤 **Author1**
+
+-- GitHub: [@githubhandle](https://github.com/businesskaniki)
+- Twitter: [@twitterhandle](https://twitter.com/kaniki7346)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/buinesskaniki)
+
+👤 **Author2**
```

Comment:
- [ ] OPTIONAL
Kindly replace the text **Author1** and **Author2**  with your respective full names, so as to make your ReadMe more professional